packages/viz/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@carto/toolkit-viz",
+  "version": "0.0.1-rc.13",
```

Comment:
Once we rebase master, we should update this to the current **0.0.1-rc.16**, also its dependencies, and run the tests****